### PR TITLE
Adding encoding detection to force the correct encoding.  fixes #23

### LINKED ISSUE #23: Possible UTF-8 conversion problems
This item in Fedora3 has this description:
https://scholarsphere.psu.edu/files/7d279232g#.VMfeyHDF_iE
7d279232g_prod
When converted, it displays as:
https://scholarsphere-qa.dlt.psu.edu/files/7d279232g#.VMfetXDF_iE
7d279232g_migrated

### PR DESCRIPTION
**Pull request by cam156**

### CODE REVIEW

**Comment by cam156** [comment]
@awead I'll remove it. I was thinking it might have good info to switch encoder on, but it did not contain what I wished for...

File: spec/integration/rdf_migration_spec.rb [hunk 1]
```diff
@@ -2,29 +2,53 @@
 
 describe "Mirating RDF terms" do
 
-  let(:mover) do
```
Comment (jcoyne) [inline spec/integration/rdf_migration_spec.rb]: You should leave `mover` up here and just define `file` as the first parameter. In each context do `let(:file) { FedoraMigrate.source.connection... }`

Comment (cam156) [inline spec/integration/rdf_migration_spec.rb]: Neat use of lazy load!

**Comment by awead** [comment]
Travis is only failing under 2.1, but this branch passes locally. Merging anyway.

**Pull request merged**